apps/charterafrica/src/payload/blocks/FeaturedNewsPost.js

```diff
@@ -0,0 +1,30 @@
+const FeaturedNewsPost = {
+  slug: "featured-news-post",
```

Comment:
If relationship field allows relating to more than one collection, we can call this just FeaturedPost and we'll reuse it for News, Research and Events collections